environments/production/group_vars/rax.yml

```diff
@@ -1,9 +1,8 @@
 ---
-environment: prod
-
 database:
   flavor: io1-15
   image: Ubuntu 14.04 LTS (Trusty Tahr) (PVHVM)
+  key_name: dirn-macbook-pro
```

Comment:
Should this be some key stored in the vault used just for creating servers and disabled after `ansible` is finished?